buildSrc/src/main/groovy/nva-orcid-client.java-common-conventions.gradle

```diff
@@ -16,11 +16,24 @@ group = 'com.github.bibsysdev'
 repositories {
     mavenCentral()
     maven { url "https://jitpack.io" }
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
 }
 
 dependencies {
 
     testImplementation libs.bundles.testing
```

Comment:
Weird that the testing bundle doesn't include the relevant stuff…

Reply:
It does. Removed excess